README.md

```diff
@@ -78,13 +98,25 @@ Work in Progress, will finish when we solidify our project structure.
 ├── package.json
 ├── pages        # Expose pages and endpoints*
 │   ├── _app.tsx
-│   ├── api      # API endpoints, 
-│   │   └── hello.ts
+│   ├── api      # API endpoints 
+│   │   ├── auth 
+│   │   │   └── [....nextauth].js
+│   │   └── news.ts
+│   ├── contact  
+│   │   └── index.tsx  
+│   ├── login  
+│   │   └── index.tsx 
+│   ├── whatsnew 
+│   │   └── index.tsx  
 │   └── index.tsx
 ├── prisma       # Prisma DB Schemas, Prisma Client, Migrations, etc.
 │   ├── migrations
 │   │   ├── 20211223003440_test
 │   │   │   └── migration.sql
+│   │   ├── 20220129153613_added_news_entity
+│   │   │   └── migration.sql
+│   │   ├── 20211223003440_test
+│   │   │   └── migration.sql
 │   │   └── migration_lock.toml
```

Comment:
⛏️ I wonder if we need to include these? Not sure it tells you much really. I would lean in favor of deleting them and just keeping /migrations

Reply:
done